util/upload-cfn-templates.py

```diff
@@ -38,6 +34,10 @@ def upload_to_s3(args, region):
                     continue
                 except ClientError:
                     pass
+            if args.dryrun:
+                key = key_path + "%s-%s.cfn.json" % (t, args.version)
+                print("Skipping upload %s to s3://%s/%s" % (template_name, bucket, key))
```

Comment:
Instead of saying skipping upload, say "If dryrun is disabled, %s will be uploaded to s3://%s/%s"